keyboards/oktiboard/readme.md

```diff
@@ -0,0 +1,10 @@
+# oktiboard
```

Comment:
This should be `keyboards/handwired/oktiboard/readme.md`, isn't it?

Reply:
yeap. it is

Reply:
Now there are 2 readmes. :)
`git rm` this.